### DISCUSSION TITLE
使用官方的V4-simple模型，检出汉字一的效果很差

### DISCUSSION THREAD

**Discussion by HHaidai** [PaddlePaddle/PaddleOCR#14066] [Q&A]
![image](https://github.com/user-attachments/assets/c3bea401-100b-4715-990c-0f854ada5f1b)
如图所示，所有一开头的框都有一定的漏检，请问这是为什么，有办法调整么？

**Comment by GreatV** [comment]
最好的办法是自己在自己的场景微调一下。请参考 https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_train/finetune.html
